core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -455,8 +456,13 @@ public int getLostWorkerCount() {
   public long getCapacityBytes() {
     long ret = 0;
     for (MasterWorkerInfo worker : mWorkers) {
-      try (LockResource r = new LockResource(worker.getUsageLock().readLock())) {
+      EnumSet<MasterWorkerInfo.LockType> lockTypes =
+          EnumSet.of(MasterWorkerInfo.LockType.USAGE_LOCK);
```

Comment:
can you make this inline where-ever used?

Reply:
I get really paranoid when I lock and unlock typing the `EnumSet` twice and fear that they don't match. I believe many people  are like this too.

`LockResource` will then be a lifesaver. I will add that in this PR.